File: mockspresso-core/src/main/java/com/episode6/hackit/mockspresso/BuildMockspresso.java
package com.episode6.hackit.mockspresso;

import com.episode6.hackit.mockspresso.internal.MockspressoBuilder;
import org.jetbrains.annotations.NotNull;

/**
 * Contains a static method to create new {@link Mockspresso.Builder}s
 */
public class BuildMockspresso {

  /**
   * @return a new {@link Mockspresso.Builder}.
   */
  public static @NotNull Mockspresso.Builder with() {
    return MockspressoBuilder.create();
  }
}

File: mockspresso-core/src/main/java/com/episode6/hackit/mockspresso/BuildMockspresso.kt
package com.episode6.hackit.mockspresso

import com.episode6.hackit.mockspresso.internal.createMockspressoBuilder

/**
 * Contains a static method to create new [Mockspresso.Builder]s
 */
object BuildMockspresso {

  /**
   * @return a new [Mockspresso.Builder].
   */
  @JvmStatic
  fun with(): Mockspresso.Builder = createMockspressoBuilder()
}
